db/seeds.rb

```diff
@@ -47,7 +47,7 @@
 
 invoice_items = []
 CSV.foreach('./app/csvs/invoice_items.csv', headers: true, header_converters: :symbol) do |data|
-  invoice_items << InvoiceItem.new(id: data[:id], item_id: data[:item_id], invoice_id: data[:invoice_id], quantity: data[:quantity], unit_price: data[:unit_price])
+  invoice_items << InvoiceItem.new(id: data[:id], item_id: data[:item_id], invoice_id: data[:invoice_id], quantity: data[:quantity], unit_price: (data[:unit_price].to_f / 100))
```

Comment:
unit_price from CSV data was whole numbers. According to spec_harness from FE, unit_price is float. Adjustment made in seed to accommodate.